besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1974,12 +1996,28 @@ private void checkPrivacyTlsOptionsDependencies() {
           "Known-clients file must be specified or CA clients must be enabled when TLS client authentication is enabled for JSON-RPC HTTP endpoint");
     }
 
+    rpcHttpTlsProtocols.retainAll(getJDKEnabledProtocols());
+    if (rpcHttpTlsProtocols.isEmpty()) {
+      throw new ParameterException(commandLine, "No valid TLS protocols specified");
```

Comment:
Could be useful to add the list of allowed protocols to the message

Reply:
Cool, I will add that